### PR TITLE
Cockpit 147

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: content/cockpit-147.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+and their passwords:
+
+![Account Expiry](images/accounts-expiry.png)
+
+Thanks to Fridolin Pokorny for this improvement!
+
+### Consistently ignore loopback traffic on all network load graphs
+
+The network load graphics on the System, Networking, and Dashboard pages
+previously sometimes accounted traffic on the loopback network device (`lo`),
```
Comment (dperpeet) [inline content/cockpit-147.md]: I think we can leave out `previously`, the sentence is much simpler without that.

File: content/cockpit-147.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+which in most cases is not interesting for administrators. The graphs have been
+harmonized to always ignore the loopback device.
```
Comment (dperpeet) [inline content/cockpit-147.md]: `harmonized` -> `changed`?

File: content/cockpit-147.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+### Fix the Kdump page in Internet Exporer
+
+This page previously produced errors when being opened in Internet Explorer.
```
Comment (dperpeet) [inline content/cockpit-147.md]: If you want to add more detail, you could say that the calls to unsupported JavaScript functions were replaced.

File: content/cockpit-147.md (hunk 1 above)
Comment (martinpitt) [inline content/cockpit-147.md]: Agreed, fixed.

File: content/cockpit-147.md (hunk 2 above)
Comment (martinpitt) [inline content/cockpit-147.md]: I simplified the sentence in a different way.

File: content/cockpit-147.md (hunk 3 above)
Comment (martinpitt) [inline content/cockpit-147.md]: Nice idea! Done.

**Comment by martinpitt** [comment]
Force-pushed to now include the [screencast](https://www.youtube.com/watch?v=q4JCoDF7XwY&feature=youtu.be) for account/password expiry, thanks Stef!

File: content/cockpit-147.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 147.
+
+### Add configuration of expiry to Accounts page
+
+The Accounts page can now display and change the expiration of user accounts
+and their passwords. Correspondingly, the login page now supports changing
+expired passwords.
```
Comment (stefwalter) [inline content/cockpit-147.md]: The login page has supported this for a while. But triggering it from the Accounts page was missing.

Comment (stefwalter) [inline content/cockpit-147.md]: "Correspondingly, the Accounts page can now force users to change their passwords on the next login."

Comment (martinpitt) [inline content/cockpit-147.md]: Thanks,  I reworded it accordingly.

**Pull request merged**